### PR TITLE
Remove vendor directory

### PR DESCRIPTION
**Pull request by thaJeztah**
### remove vendor directory

This project is used as a library and doesn't produce binaries,
in which case convention is to not use vendoring.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,7 +1,7 @@
 module github.com/containerd/go-cni
 
 require (
-	github.com/containernetworking/cni v0.8.0
+	github.com/containernetworking/cni v0.8.1
```
Comment (mikebrow) [inline go.mod:4]: not sure if it makes sense to move up to 0.8.1.. crio and k/k are at 0.8.0 and all I see there is a "tightening" of the plugin finding logic with no explanation

**Review by mikebrow — COMMENTED** [review]
the remove vendor directory commit LGTM

File: go.mod (hunk 1 above)
Comment (thaJeztah) [reply go.mod:4]: Let me move the last commit from this PR

**Comment by thaJeztah** [comment]
@mikebrow updated; now only removing the vendor directory in this PR

**Review by dims — APPROVED** [review]
LGTM

thanks for just limiting this to vendor/

**Pull request merged**